src/plugins/plugin-gmaps/carto-layer-gmaps.js

```diff
@@ -1,5 +1,5 @@
 import Promise from 'bluebird';
-import cartoService from './carto-layer-service';
+import cartoService from '../../services/carto-layer-service';
```

Comment:
Should we use some import resolver here? I know that you didn't introduce this error but it would be nice to solve it

Reply:
Babel supports this I will add it tomorrow.